### ISSUE TITLE
create a React Catch All router 

### ISSUE THREAD

**Issue by dpickett** [LaunchAcademy/generator-engage#68]
we need a catch all router like so:

```javascript
import { fileURLToPath } from "url";
import path, { dirname } from "path";
import express from "express";

const router = new express.Router();
const currentPath = dirname(fileURLToPath(import.meta.url));

router.get("*", (req, res) => {
  res.sendFile(path.join(currentPath, "../public/index.html"));
});

export default router;
```

**Comment by dpickett** [comment]
fairly large concern with using an explicit allow list for client side routes vs. a catch all - 

Locally, students will be running webpack-dev-server, so react router will work as expected. It will be in production where it breaks. Knowing our students (and myself), this is a detail that’s easy to forget :-/

wdyt @bkincart @nwalberts ?

**Comment by dpickett** [comment]
Brianna, Nick and I spoke and we've decided to actually run the application in development in the other direction. We'll use webpack middleware in dev to serve out the express endpoints allowing us to provide for a higher fidelity dev experience for students.

**Comment by dpickett** [comment]
fixed in #88
